goal_src/jak2/build/all_objs.json

```diff
@@ -0,0 +1,2252 @@
+["ocean-mid",                                  "ocean-mid",                  3, ["ENGINE", "GAME"], "engine/gfx/ocean"],
+["ocean-transition",                           "ocean-transition",           3, ["ENGINE", "GAME"], "engine/gfx/ocean"],
+["ocean-near",                                 "ocean-near",                 3, ["ENGINE", "GAME"], "engine/gfx/ocean"],
+["minimap",                                    "minimap",                    3, ["ENGINE", "GAME"], "engine/ui"],
+["bigmap-data",                                "bigmap-data",                3, ["ENGINE", "GAME"], "engine/ui"],
+["bigmap",                                     "bigmap",                     3, ["ENGINE", "GAME"], "engine/ui"],
+["eye",                                        "eye",                        3, ["ENGINE", "GAME"], "engine/gfx"],
+["glist-h",                                    "glist-h",                    3, ["ENGINE", "GAME"], "engine/util"],
+["glist",                                      "glist",                      3, ["ENGINE", "GAME"], "engine/util"],
+["anim-tester",                                "anim-tester",                3, ["ENGINE", "GAME"], "engine/debug"],
+["viewer",                                     "viewer",                     3, ["ENGINE", "GAME"], "engine/debug"],
+["part-tester",                                "part-tester",                3, ["ENGINE", "GAME"], "engine/debug"],
+["editable-h",                                 "editable-h",                 3, ["ENGINE", "GAME"], "engine/debug/nav"],
+["editable",                                   "editable",                   3, ["ENGINE", "GAME"], "engine/debug/nav"],
+["editable-player",                            "editable-player",            3, ["ENGINE", "GAME"], "engine/debug/nav"],
+["mysql-nav-graph",                            "mysql-nav-graph",            3, ["ENGINE", "GAME"], "engine/debug/nav"],
+["nav-graph-editor",                           "nav-graph-editor",           3, ["ENGINE", "GAME"], "engine/debug/nav"],
+["sampler",                                    "sampler",                    3, ["ENGINE", "GAME"], "engine/sound"],
+["default-menu",                               "default-menu",               3, ["ENGINE", "GAME"], "engine/debug"],
+["los-control-h",                              "los-control-h",              3, ["GAME", "COMMON"], "levels/common"],
+["airlock",                                    "airlock",                    3, ["GAME", "COMMON"], "levels/common"],
+["water-anim",                                 "water-anim",                 3, ["GAME", "COMMON"], "levels/common"],
+["blocking-plane",                             "blocking-plane",             3, ["GAME", "COMMON"], "levels/common"],
+["idle-control",                               "idle-control",               3, ["GAME", "COMMON"], "levels/common"],
+["dark-eco-pool",                              "dark-eco-pool",              3, ["GAME", "COMMON"], "levels/common"],
```

Comment:
all these could be common_obs, I think, except idle-control which could go in `game`